### PR TITLE
Update postman

### PR DESCRIPTION
**Pull request by TimCsaky**
# Description

supports [SHOWCASE-887](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-887)

updated postman collection for CDOGS to use CDOGS API V2
This involved adding the new /template and /render endpoints

The link to download the postman collections is shown in step 4 of the api access form.
The new collection files have both been added to the public directory. (/files)

## Types of changes

Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

I need to test this collections works in Postman

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-k8vopl-tools.pathfinder.gov.bc.ca/dashboard?id=nr-get-token-pr-171

**Comment by TimCsaky** [comment]
ah yes I dont know. It makes sense to have one source for the postman collections.
but those two links to download a collection for each api do look good.

File: app/frontend/src/utils/commonServices.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ export const CommonServiceList = Object.freeze([{
   type: kc,
   shortName: 'common-document-generation',
   name: 'Common Document Generation Service',
-  apiDocLink: 'https://cdogs-master-idcqvl-prod.pathfinder.gov.bc.ca/api/v1/docs',
+  apiDocLink: 'https://cdogs-master-idcqvl-dev.pathfinder.gov.bc.ca/api/v2/docs',
```
Comment (TimCsaky) [inline app/frontend/src/utils/commonServices.js:53]: this should be a link to the PROD api docs but I believe it's currently not public.
I'll change this if possible

**Comment by loneil** [comment]
@TimCsaky postman download links don't work

**Comment by TimCsaky** [comment]
oops. sorry.. wtf relative paths dont seem to work either for the pr branch. 
I'm guessing an alternative is to get the 'base url' using js or the node 'url' library

**Comment by TimCsaky** [comment]
well i can confirm the requests to cdogs api v2 all work. pretty awesome. and they show up on dgrsc performance dashboard

**Pull request merged**